src/performance/network.ts

```diff
@@ -18,13 +19,13 @@ export function compute(entry: PerformanceResourceTiming): void {
     state.push({
         url: entry.name,
         data: {
-            start: Math.round(entry.startTime),
-            duration: Math.round(entry.duration),
-            size: entry.transferSize,
+            start: "startTime" in entry ? Math.round(entry.startTime) : -1,
```

Comment:
Why don't you just push null here and then not have the extra complexity in other file?

Reply:
Thinking more on this. Leaving this comment open.

---
In reply to: [368666995](https://github.com/microsoft/clarity-js/pull/166#discussion_r368666995) [](ancestors = 368666995)

Reply:
Simplified the code and using null for null, and only for fields where it makes sense.

---
In reply to: [368810644](https://github.com/microsoft/clarity-js/pull/166#discussion_r368810644) [](ancestors = 368810644,368666995)